magda-ckan-connector/src/Ckan.ts

```diff
@@ -1,10 +1,14 @@
 import AsyncPage, { forEachAsync } from '@magda/typescript-common/dist/AsyncPage';
 import CkanUrlBuilder from './CkanUrlBuilder';
 import formatServiceError from '@magda/typescript-common/dist/formatServiceError';
-import { ConnectorSource } from '@magda/typescript-common/dist/JsonConnector';
+import { 
+    ConnectorSource
+} from '@magda/typescript-common/dist/JsonConnector';
 import retry from '@magda/typescript-common/dist/retry';
 import * as request from 'request';
 import * as URI from 'urijs';
+import * as moment from 'moment'
+import {DatasetContainer} from 'aspect-templates/../../magda-typescript-common/src/JsonConnector'
```

Comment:
This path looks wrong.  I think you want `@magda/typescript-common/dist/JsonConnector`.